### PR TITLE
Fixed support long data for HTTP

### LINKED ISSUE #161: Bug with long data in HTTP
```java
  @Test
  void shouldReturnSingleResponseWithSimpleLongDataRequest() {
    String data = new String(new char[170]);
    StepVerifier.create(service.one(data)).expectNext("Echo:" + data).verifyComplete();
  }
```
This test works well via the WebSocket protocol but it will fail via the HTTP protocol. The HTTP acceptor receives 2 requests instead of 1 and then it tries to emit 2 responses to the publisher. Maybe the problem is chunk support.

NOTE: there is no problem in the client-sdk, I've received the same behavior using the postman and the curl.

### PR DESCRIPTION
**Pull request by segabriel**
closes #161

### CODE REVIEW

File: ClientSdk/src/main/java/io/scalecube/gateway/clientsdk/http/HttpClientTransport.java [hunk 1]
```diff
@@ -86,15 +86,15 @@ public HttpClientTransport(
                                   // no-op
                                 })
                             .receive()
+                            .aggregate()
```
Comment (artem-v) [inline ClientSdk/src/main/java/io/scalecube/gateway/clientsdk/http/HttpClientTransport.java]: With out aggregate we would not getting fullt_http_request? Is that what u saying?

File: ClientSdk/src/main/java/io/scalecube/gateway/clientsdk/http/HttpClientTransport.java [hunk 2]
```diff
@@ -86,15 +86,15 @@ public HttpClientTransport(
...
                             .map(ByteBuf::retain)
                             .map(
                                 content ->
                                     handleResponseContent(
                                         httpResponse, content, request.qualifier()))
                             .map(
                                 response ->
-                                    enrichResponse(response, httpResponse.responseHeaders()))
-                            .singleOrEmpty()));
```
Comment (artem-v) [inline ClientSdk/src/main/java/io/scalecube/gateway/clientsdk/http/HttpClientTransport.java]: `singleOrEmpty` removed, can u explain why?

File: Http/src/main/java/io/scalecube/gateway/http/GatewayHttpAcceptor.java [hunk 3]
```diff
@@ -60,10 +60,11 @@
 
     return httpRequest
         .receive()
+        .aggregate()
```
Comment (artem-v) [inline Http/src/main/java/io/scalecube/gateway/http/GatewayHttpAcceptor.java]: Again, u added aggregate, u mean like with out it we would not getting full_http_request which could possibly come in many chunks of byte bufs ?

File: ClientSdk/src/main/java/io/scalecube/gateway/clientsdk/http/HttpClientTransport.java (hunk 1 above)
Comment (segabriel) [reply ClientSdk/src/main/java/io/scalecube/gateway/clientsdk/http/HttpClientTransport.java]: I added simple test and checked it (#161) and yeah without `aggregate` it splits by chunks.

File: ClientSdk/src/main/java/io/scalecube/gateway/clientsdk/http/HttpClientTransport.java (hunk 2 above)
Comment (segabriel) [reply ClientSdk/src/main/java/io/scalecube/gateway/clientsdk/http/HttpClientTransport.java]: `singleOrEmpty` it's for `flux` to cast to `mono`, I added `aggregate` and it turned into `mono`.

File: Http/src/main/java/io/scalecube/gateway/http/GatewayHttpAcceptor.java (hunk 3 above)
Comment (segabriel) [reply Http/src/main/java/io/scalecube/gateway/http/GatewayHttpAcceptor.java]: Yes, the same as above.

**Pull request merged**